src/app/producer-shelf/components/producer-shelf/producer-shelf.component.html

```diff
@@ -5,12 +5,18 @@
           [title]="actionPanel.name | uppercase"
           [actionRankForOnAirPart]="actionRankForOnAirPart"
           [actions]="actionPanel.actions"
-          (executeAction)="executeAction($event)"
+          [actionIdsWithPiecesOnAir]="actionIdsWithPiecesOnAir"
+          (executeAction)="executeOrStopActionBasedOnPiecesOnAir($event)"
         ></sofie-tv2-action-panel>
         <sofie-virtual-keyboard [keyBindings]="keyBindings" [keystrokes]="keystrokes"></sofie-virtual-keyboard>
     </div>
 
     <div class="sidebar">
-        <sofie-tv2-action-panel title="STATIC BUTTONS" [actions]="staticButtonActions" (executeAction)="executeAction($event)" [isScrollable]="false"></sofie-tv2-action-panel>
+        <sofie-tv2-action-panel
+          title="STATIC BUTTONS"
```

Comment:
```suggestion
          [title]="'STATIC BUTTONS'"
```

Just a suggestion to put the input notation on :)

Reply:
Why should it be dynamic, when the content is static? :)